tidb-cloud/terraform-use-cluster-resource.md

```diff
@@ -7,17 +7,17 @@ summary: Learn how to use the cluster resource to create and modify a TiDB Cloud
 
 You can learn how to create and modify a TiDB Cloud cluster with the `tidbcloud_cluster` resource in this document.
 
-In addition, you will also learn how to get the necessary information with the `tidbcloud_project` and `tidbcloud_cluster_spec` data sources.
+In addition, you will also learn how to get the necessary information with the `tidbcloud_projects` and `tidbcloud_cluster_specs` data sources.
 
 ## Prerequisites
 
 - [Get TiDB Cloud Terraform Provider](/tidb-cloud/terraform-get-tidbcloud-provider.md).
 
-## Get project IDs using the project data source
+## Get project IDs using the projects data source
```

Comment:
```suggestion
## Get project IDs using the `tidbcloud_projects` data source
```

Reply:
if using this style, update other data source/resource too.